src/components/ReactGoogleMaps.js

```diff
@@ -244,16 +256,39 @@ export class ReactGoogleMaps extends Component {
             }}
             center={{ lat: this.state.currlat, lng: this.state.currlon }}
           >
+            <TypeToggle/>
+
+          {/* FilteredTaps */}
+
+          {this.props.phlaskType === PHLASK_TYPE_WATER
+            ? <WaterFilter/>
+            : <FoodFilter/>
+          }
 
-            <Filter/>
+          {/* Issue: MapMarkers won't render when placed inside container? */}
```

Comment:
Is this still an issue? Looks like the markers are all rendering properly in the test site

Reply:
For some reason when I place the Marker components inside a parent element to conditionally render with with their respective filters, the filters render but the markers don't. They show up in devtools but not on the map.
It's not a breaking issue, I solved it by rendering the Markers separate from their filter (ideally they'd be together). It was mostly just a comment for me to come back to later haha